.cicd/Jenkinsfile

```diff
@@ -0,0 +1,151 @@
+pipeline {
```

Comment:
Is it possible to include a comment describing what this file does?  I don't know the Jenkins syntax used here and am only guessing at what's going on.  Thanks.

Reply:
Looks good, thanks.